docs/index.html

```diff
@@ -59,13 +59,36 @@ <h3 class="card-title mb-3">Stay up to date</h3>
     <!--End mc_embed_signup-->
 
     <p class="text-muted font-italic small">* We never spam and you can opt out at any time.</p>
-    
   </div>
-
-  <p class="my-4">
-    <a href="{{ site.tw_url }}" target="_blank" rel="noopener" class="btn btn-outline-secondary"><i class="fa fa-fw fa-twitter"></i></a>
-    <a href="{{ site.fb_url }}" target="_blank" rel="noopener" class="btn btn-outline-secondary"><i class="fa fa-fw fa-facebook"></i></a>
-    <a href="{{ site.gh_url }}" target="_blank" rel="noopener" class="btn btn-outline-secondary"><i class="fa fa-fw fa-github"></i></a>
-  </p>
 </div>
 
+<footer class="text-white bg-dark">
+    <div class="d-flex flex-row justify-content-center py-5">
+      <div class="col-3 text-center">
+        <h6 class="text-uppercase">Connect with us</h6>
+        <p class="my-4 text-center">
+            <a href="{{ site.tw_url }}" target="_blank" rel="noopener" class="btn btn-outline-light"><i class="fa fa-fw fa-twitter"></i></a>
+            <a href="{{ site.fb_url }}" target="_blank" rel="noopener" class="btn btn-outline-light"><i class="fa fa-fw fa-facebook"></i></a>
+            <a href="{{ site.gh_url }}" target="_blank" rel="noopener" class="btn btn-outline-light"><i class="fa fa-fw fa-github"></i></a>
+        </p>
+        <a href="https://medium.com/code-for-africa"><button class="btn btn-outline-light bt-md" style="width:150px;">Read More</button></a>
+      </div>
+      <div class="col-3 text-center border-light border-right border-left">
+          <h6 CLASS="text-uppercase">About sensors.AFRICA</h6>
+          <p class="pt-1 small">
+             sensors.AFRICA is a pan-African citizen science initiative that uses sensors to monitor air,
+               water and sound pollution to give citizens actionable information about their cities
+          </p>
+          <p class="small">All content released under a <a href="https://creativecommons.org/licenses/by/4.0/" target="_blank" rel="noopener">Creative Commons 4 Attribution license</a>.</p>
+         
+      </div>
+      <div class="col-3 text-center">
+        <p class="pt-4 small">This initiative was seed-funded by innovateAFRICA, and is 
+          being incubated by Code for Africa
+        </p>
+        <img src="/images/partnerslogoswhite.png" height="80px" alt="Partner's Logos">
```

Comment:
Please link this to https://medium.com/code-for-africa/tagged/innovateafrica and let it open in a new tab.